src/app/src/pages/redteam/report/components/TestSuites.tsx

```diff
@@ -32,13 +34,34 @@ interface TestSuitesProps {
   plugins: RedteamPluginObject[];
 }
 
+const getRiskScoreColor = (riskScore: number): string => {
+  if (riskScore >= severityRiskScores[Severity.Critical]) {
+    return '#d32f2f'; // Red for critical
+  } else if (riskScore >= severityRiskScores[Severity.High]) {
+    return '#f57c00'; // Orange for high
+  } else if (riskScore >= severityRiskScores[Severity.Medium]) {
+    return '#fbc02d'; // Yellow for medium
+  } else {
+    return '#388e3c'; // Green for low
+  }
+};
+
 const getSubCategoryStats = (
   categoryStats: Record<string, { pass: number; total: number; passWithFilter: number }>,
   plugins: RedteamPluginObject[],
 ) => {
   const subCategoryStats = [];
   for (const subCategories of Object.values(riskCategories)) {
     for (const subCategory of subCategories) {
+      const attackSuccessRate = categoryStats[subCategory]
+        ? (
+            ((categoryStats[subCategory].total - categoryStats[subCategory].pass) /
+              categoryStats[subCategory].total) *
+            100
+          ).toFixed(1) + '%'
```

Comment:
Extreme nitpick:
```suggestion
            ((categoryStats[subCategory].total - categoryStats[subCategory].pass) /
              categoryStats[subCategory].total) *
            100
          ).toLocaleString(undefined, {style: 'percent', minimumFractionDigits:1})
```

Also, why is this fixed to 1 decimal spot? isn't 2 pretty standard?